src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1612,6 +1613,11 @@ internal void AssertMemberExposure(Symbol member, bool forDiagnostics = false)
                 // Backing fields for field-like events are not added to the members list.
                 member = e;
             }
+            else if (member is SynthesizedExtensionMarker)
+            {
+                RoslynDebug.AssertOrFailFast(member.ContainingType.IsExtension);
```

Comment:
>RoslynDebug.AssertOrFailFast(member.ContainingType.IsExtension); [](http://example.com/codeflow?start=16&length=64)

Is this code path reachable?

Reply:
Yes, many tests reach this

Reply:
> Yes, many tests reach this

I am curious why would that be the case. My understanding is that ```SynthesizedExtensionMarker``` is not supposed to be among members of the type.